io.github.qtox.qTox.json

```diff
@@ -0,0 +1,100 @@
+{
+	"app-id": "io.github.qtox.qTox",
+	"runtime": "org.kde.Platform",
+	"sdk": "org.kde.Sdk",
+	"runtime-version": "5.10",
+	"command": "qtox",
+	"rename-icon": "qtox",
+	"rename-desktop-file": "qtox.desktop",
+	"rename-appdata-file": "qTox.appdata.xml",
+	"finish-args": [
+		"--share=network",
+		"--socket=pulseaudio",
+		"--socket=wayland",
+		"--socket=x11",
+		"--share=ipc",
+		"--filesystem=host",
+		"--device=all"
+	],
+	"build-options": {
+		"cflags": "-O3 -DSQLITE_HAS_CODEC",
+		"cxxflags": "-O3"
+	},
+	"cleanup": [
+		"/include",
+		"/lib/pkgconfig",
+		"/share/man"
+	],
+	"modules": [
+		{
+			"name": "sqlcipher",
+			"rm-configure": true,
+			"config-opts": [
+				"--enable-tempstore=yes",
+				"--disable-tcl"
+			],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/sqlcipher/sqlcipher",
+					"tag": "v3.4.2",
+					"commit": "c6f709fca81c910ba133aaf6330c28e01ccfe5f8"
+				},
+				{
+					"type": "script",
+					"dest-filename": "autogen.sh",
+					"commands": [
+						"AUTOMAKE=\"automake --foreign\" autoreconf -vfi"
+					]
+				}
+			]
+		},
+		{
+			"name": "libsodium",
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/jedisct1/libsodium",
+					"tag": "1.0.16",
+					"commit": "675149b9b8b66ff44152553fb3ebf9858128363d"
+				}
+			]
+		},
+		{
+			"name": "libqrencode",
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/fukuchi/libqrencode",
+					"tag": "v4.0.0", 
+					"commit": "07f3c5d4bf9136711422cc7dbf28aff469da220a"
+				}
+			]
+		},
+		{
+			"name": "c-toxcore",
+			"buildsystem": "cmake",
```

Comment:
Nitpick that can be fixed later: `s/cmake/cmake-ninja` as it might cut some compilation time.

Reply:
fixed